### PR TITLE
Mocha 6 'stats' support

### PR DESCRIPTION
**Pull request by fargies**
Hello,

I had an issue running mocha-multi with mocha 6.1.4, it seems that latest mocha versions adds a 'stats' property on runner objects.

The command I was running:
```bash
multi="mocha-notifier-reporter=- spec=-" mocha -R mocha-multi
```

Thrown error:
```
.../node_modules/mocha-notifier-reporter/index.js:19
	stats.total = runner.total;
	            ^

TypeError: Cannot set property 'total' of undefined
    at notify (.../node_modules/mocha-notifier-reporter/index.js:19:14)
    at NotifyReporter.<anonymous> (.../node_modules/mocha-notifier-reporter/index.js:44:3)
    at EventEmitter.emit (events.js:182:13)
    at withReplacedStdout (.../node_modules/mocha-multi/mocha-multi.js:197:14)
    at withReplacedStdout (.../node_modules/mocha-multi/mocha-multi.js:146:12)
    at Runner.runner.on.eventArgs (.../node_modules/mocha-multi/mocha-multi.js:196:7)
    at Runner.emit (events.js:187:15)
    at .../node_modules/mocha/lib/runner.js:903:12
    at done (.../node_modules/mocha/lib/runner.js:761:7)
    at next (.../node_modules/mocha/lib/runner.js:728:16)
    at done (.../node_modules/mocha/lib/runner.js:761:7)
    at next (.../node_modules/mocha/lib/runner.js:732:14)
    at Runner.hookErr (.../node_modules/mocha/lib/runner.js:575:7)
    at Runner.uncaught (.../node_modules/mocha/lib/runner.js:857:19)
    at process.uncaught (.../node_modules/mocha/lib/runner.js:887:10)
    at process.emit (events.js:182:13)
    at process._fatalException (internal/bootstrap/node.js:491:27)
```

This patch solves the issue, ran `npm run test` on it

### CODE REVIEW

**Comment by Avyiel** [comment]
Would love for this to be merged 👍

File: mocha-multi.js [hunk 1]
```diff
@@ -71,7 +71,7 @@ function convertSetup(reporters) {
       try {
         setup = setup.concat(convertSetup(JSON.parse(fs.readFileSync(reporters[reporter]))));
       } catch (e) {
-        bombOut('bad_file', reporters[reporter], e);
+        bombOut('bad_file', reporters[reporter], e.message);
```
Comment (glenjamin) [inline mocha-multi.js]: This change seems unrelated to the reported issue - could you explain why it's included?

Comment (fargies) [reply mocha-multi.js]: This modification is related to an internal change in Node that has happened between v10.15.3 and v12.7.0.

It seems that displaying 'e' dumps a complete stack trace with the later version, which was not the case before, and this makes the tests fail.

This line goes with the 'Error: ' modification, using only the message part which seems to be what the tests expects.

Indeed it is not directly related to the 'stats' modification, but this was required for the CI part and to run the unit-tests

Comment (glenjamin) [reply mocha-multi.js]: Aha, that makes sense - thanks.

**Comment by buddyp450** [comment]
Am running into this today, any chance we could get an `npm publish`?

**Comment by glenjamin** [comment]
Published as `mocha-multi@1.1.1`
